### PR TITLE
README.md updates, main branch

### PR DESCRIPTION
**Pull request by johnkerl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,29 @@
 <a href="https://tiledb.com"><img src="https://github.com/TileDB-Inc/TileDB/raw/dev/doc/source/_static/tiledb-logo_color_no_margin_@4x.png" alt="TileDB logo" width="400"></a>
 
-[![TileDB-SOMA CI](https://github.com/single-cell-data/TileDB-SOMA/actions/workflows/ci.yml/badge.svg)](https://github.com/single-cell-data/TileDB-SOMA/actions/workflows/ci.yml)
+[![TileDB-SingleCell CI](https://github.com/single-cell-data/TileDB-SingleCell/actions/workflows/ci.yml/badge.svg)](https://github.com/single-cell-data/TileDB-SingleCell/actions/workflows/ci.yml)
```
Comment (bkmartinjr) [inline README.md:3]: shouldn't the repo name be `TileDB-SOMA`?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
 
-# TileDB-SOMA
+# TileDB-SingleCell
```
Comment (bkmartinjr) [inline README.md:5]: s/TileDB-SingleCell/TileDB-SOMA/g

File: README.md [hunk 3]
```diff
@@ -1,11 +1,29 @@
...
 
 A C++ library with APIs in Python and R for efficient storage and retrieval of single-cell data using [TileDB Embedded][tiledb].
 
+Note this branch, `main`, implements the [newer specfication](https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md).  Please also see the `main-old` branch which implements the [older specification](https://github.com/single-cell-data/TileDB-SOMA/blob/main-old/spec/specification.md).
+
+# Language bindings and installation instructions
+
+* [Python](https://github.com/single-cell-data/TileDB-SOMA/tree/main/apis/python)
+* [R](https://github.com/single-cell-data/TileDB-SOMA/tree/main/apis/r)
+* [API documentation](https://tiledb-inc-tiledb-soma.readthedocs-hosted.com/en/latest/)
```
Comment (aaronwolen) [inline README.md:15]: I think it might be confusing to link the original API docs from `main`

File: README.md [hunk 4]
```diff
@@ -1,11 +1,29 @@
...
+  * Note: these document the `main-old` branch; `main` docs are a WIP.
+
+# Issues and contacts
```
Comment (bkmartinjr) [inline README.md:18]: Recommend an explicit section for reporting security issues, as you usually do not want those filed as an open issue.

Here is what we use for most of our organization's repos:

```
# Reporting Security Issues
Please note: If you believe you have found a security issue, please responsibly disclose it by contacting [security@????.com](mailto:security@????.com).
```

File: README.md [hunk 5]
```diff
@@ -1,11 +1,29 @@
...
 
-[![TileDB-SOMA CI](https://github.com/single-cell-data/TileDB-SOMA/actions/workflows/ci.yml/badge.svg)](https://github.com/single-cell-data/TileDB-SOMA/actions/workflows/ci.yml)
+[![TileDB-SingleCell CI](https://github.com/single-cell-data/TileDB-SingleCell/actions/workflows/ci.yml/badge.svg)](https://github.com/single-cell-data/TileDB-SingleCell/actions/workflows/ci.yml)
 
-# TileDB-SOMA
+# TileDB-SingleCell
 
 A C++ library with APIs in Python and R for efficient storage and retrieval of single-cell data using [TileDB Embedded][tiledb].
 
+Note this branch, `main`, implements the [newer specfication](https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md).  Please also see the `main-old` branch which implements the [older specification](https://github.com/single-cell-data/TileDB-SOMA/blob/main-old/spec/specification.md).
```
Comment (aaronwolen) [inline README.md:9]: Wording suggestion:

```suggestion
Note this branch, `main`, implements the [updated specfication](https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md).  Please also see the `main-old` branch which implements the [original specification](https://github.com/single-cell-data/TileDB-SOMA/blob/main-old/spec/specification.md).
```

**Review by bkmartinjr — CHANGES_REQUESTED** [review]
Couple of minor issues:
* repo name needs to be updated
* suggestion only: add a more discrete contact method for security issues

**Review by aaronwolen — CHANGES_REQUESTED** [review]
Than

File: README.md (hunk 1 above)
Comment (johnkerl) [reply README.md:3]: yes @bkmartinjr must be a bad merge 👀

Comment (bkmartinjr) [reply README.md:3]: whew :exclamation:

File: README.md [hunk 6]
```diff
@@ -1,11 +1,27 @@
 <a href="https://tiledb.com"><img src="https://github.com/TileDB-Inc/TileDB/raw/dev/doc/source/_static/tiledb-logo_color_no_margin_@4x.png" alt="TileDB logo" width="400"></a>
 
-[![TileDB-SOMA CI](https://github.com/single-cell-data/TileDB-SOMA/actions/workflows/ci.yml/badge.svg)](https://github.com/single-cell-data/TileDB-SOMA/actions/workflows/ci.yml)
+[![TileDB-SingleCell CI](https://github.com/single-cell-data/TileDB-SingleCell/actions/workflows/ci.yml/badge.svg)](https://github.com/single-cell-data/TileDB-SingleCell/actions/workflows/ci.yml)
```
[the remark this answers is not in the document]
Comment (johnkerl) [reply README.md]: @aaronwolen bad merge as noted above in comments from @bkmartinjr

File: README.md (hunk 4 above)
Comment (johnkerl) [reply README.md:15]: thanks @bkmartinjr ! let me find out our internal policy/POC for this

Comment (johnkerl) [reply README.md:15]: updated

**Review by bkmartinjr — APPROVED** [review]
LGTM

**Pull request merged**